test/backend_test.in.cpp

```diff
@@ -5233,8 +5243,8 @@ NGRAPH_TEST(${BACKEND_NAME}, sigmoid_bprop_n1c1h4)
     auto handle = backend->compile(func);
     handle->call_with_validate({result}, {a, b});
 
-    vector<float> expected{0.196612f, 0.0176627f, 0.196612f, 0.0176627f};
-    EXPECT_TRUE(test::all_close(expected, read_vector<float>(result)));
+    vector<float> expected{0.1966119f, 0.01766273f, 0.1966119f, 0.01766273f};
```

Comment:
Rather than using magic values, let's compute the correct values. sigma=1/(1+exp(-x)) and value=sigma(1-sigma).